languages/julia/demos/scripts/run_demo.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --time=00-00:30:00
+#SBATCH --ntasks-per-node=4
+
+# RUN REQUIREMENTS: You must do the following to actual use this script
+#    - add the line
+#           #SBATCH --account=<project_handle>
+#      with an appropriate resource allocation in place of <project_handle>
+#    - create a conda environment named 'py-jl-mpi' with the following
+#      python modules installed
+#        * python=3
+#        * numpy
+#        * mpi4py (installed against the same mpi module that is loaded below)
+#        * julia
+#    - an install of julia with the environment given by ./Project.toml
+#      (relative to this script) instantiated and PyCall installed in
+#      the default julia environment
+# See the README.md in this directory for instructions on setting this up.
+
+module purge
+
+module load conda/2019.10
```

Comment:
This version of conda is no longer available on the default module set. You can either use:

module load conda

or

module use /nopt/nrel/apps/modules/centos74/modulefiles
module load conda/5.3